community/libzip.hpkg

```diff
@@ -0,0 +1,66 @@
+(use ../prelude)
+(import ../core)
+(use ./cmake)
+(use ./zlib)
+
+(defsrc libzip-src
+  :file-name
+   "libzip-1-6-1.tar.gz"
+  :url
+   "https://github.com/nih-at/libzip/archive/rel-1-6-1.tar.gz"
+  :hash
+   "sha256:4f4e1991bf6b592e3d602d6b7b753c53ddadb2d76fca27d58120b8fadef7f1bf")
+
+(defpkg libzip
+  :builder
+   (fn []
+     (os/setenv "PKG_CONFIG_PATH"
+                (join-pkg-paths ":" "/lib/pkgconfig"
+                                 [zlib]))
+
+     (os/setenv "PATH"
+                (join-pkg-paths ":" "/bin"
+                                [core/awk
+                                 core/coreutils
+                                 core/pkgconf
+                                 core/gcc
+                                 core/grep
+                                 core/make
+                                 core/sed
+                                 cmake]))
+
+     (os/setenv "CFLAGS"
+                (string *default-cflags*
+                        " "
+                        "-I" (zlib :path) "/include"
+                        " "
+                        (sh/$<_ pkg-config --cflags zlib)))
+     (os/setenv "LDFLAGS"
+                (string *default-ldflags*
+                        " "
+                        "-L" (zlib :path) "/lib"
+                        " "
```

Comment:
I inserted `"-Wl,--enable-new-dtags"` with appropriate spaces padding around here to get the resulting executables and shared objects to use RUNPATH.  One example of this type of thing here: https://github.com/andrewchambers/hpkgs/blob/master/community/proot.hpkg#L40

Reply:
I updated with your suggestion, and it now uses RUNPATH instead of RPATH